linux/fedora-rpm/entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+if [ ! -d "audacity" ]
+then
+    git clone https://github.com/audacity/audacity
+fi
+
+tar -czf /root/rpmbuild/SOURCES/audacity.tar.gz audacity
+
+ls /root/rpmbuild/SOURCES/
+
+rpmbuild -bs /root/rpmbuild/SPEC/audacity.spec
+
+mock -r audacity /root/rpmbuild/SRPMS/*
```

Comment:
Do you really need to run mock inside a CI container?

Reply:
You can just install  run `dnf build-dep audacity.spec` inside the container with needed version of Fedora. But maybe CI containers may be based on a too old version of Fedora, I don't know.

Reply:
Docker helps a lot with managing reproducible, clean environments.